package.json

```
@@ -20,14 +20,15 @@
   "scripts": {
     "test:node": "mocha test/test-*.spec.js",
     "test:browser": "playwright-test test/test-*.js",
-    "test:react-native": "jest test/test-lib.react-native.spec.js",
+    "test:react-native": "npm test --prefix test/react-native",
```

We can fix windows CI errors with this.

```suggestion
    "test:react-native": "cd test/react-native && npm test",
```